edb/testbase/server.py

```diff
@@ -643,8 +644,17 @@ class DatabaseTestCase(ClusterTestCase, ConnectedTestCaseMixin):
     TEARDOWN_METHOD = None
 
     # Some tests may want to manage transactions manually,
-    # in which case ISOLATED_METHODS will be False.
-    ISOLATED_METHODS = True
+    # or affect non-transactional state, in which case
+    # TRANSACTION_ISOLATION must be set to False
+    TRANSACTION_ISOLATION = True
+
+    # By default, tests from the same testsuite may be ran
+    # in parallel in several test worker processes.  However,
+    # certain cases might exhibit pathological locking behavior,
+    # or are parallel-unsafe altogether, in which case
+    # PARALLELISM_GRANULARITY must be set to 'database' or 'system'.
```

Comment:
This comment doesn't actually explain what "database" or "system" do.

Reply:
Added an explanation.